angrmanagement/ui/widgets/qinst_annotation.py

```diff
@@ -21,6 +21,14 @@ class QInstructionAnnotation(QGraphicsSimpleTextItem):
     foreground_color = None
     addr = None
     _config = Conf
+        
+    @staticmethod
+    def get_disasm_view() -> 'DisassemblyView':
+        return GlobalInfo.main_window.workspace.view_manager.first_view_in_category("disassembly")
```

Comment:
This isn't right - if there are two disasm views open, the annotations in one of them can return the other one. You need to explicitly pass in the parent disassembly view as a constructor parameter.

Reply:
I move `disasm_view` ref to the container `QBlockAnnotations`.